### ISSUE TITLE
"Which annotations can i view with the tomogram" documentation

### ISSUE THREAD

**Issue by junxini** [chanzuckerberg/cryoet-data-portal#465]
Documentation to explain "Which annotations can I view with the tomogram" located above the annotation table. this will be a link to a relevant section to our documentation site ([see design here](https://www.figma.com/file/gwX2PjfGPfMrbEWMMJq6rW/H1-2024---Annotations-Table-Update?type=design&node-id=561%3A171504&mode=dev))

**Comment by junxini** [comment]
will not be added until annotations can be viewed in Neuroglancer

**Comment by junxini** [comment]
@dgmccart #462 is dependent on having documentation for this. Do you have bandwidth to pick up this documentation work in the next 3 weeks?

**Comment by dgmccart** [comment]
Yes, that timeline works for me. Where can I find the information on the implementation to be able to write the documentation?

**Comment by junxini** [comment]
> Yes, that timeline works for me. Where can I find the information on the implementation to be able to write the documentation?

Utz will be flagging which annotations are considered default, @uermel could you help provide more context?

**Comment by uermel** [comment]
I've already selected the default annotations for all the currently annotated datasets (`is_visualization_default` metadata field). There is no definitive rule for which annotations are displayed by default. I tried including as many as possible, provided they don't overlap and occlude each other (e.g. when Cytoplasm is annotated as a whole it would occlude other annotations included within, such as protein picks). When there was a ground truth and predicted annotation I selected the ground truth to be the annotation defualt.

**Comment by dgmccart** [comment]
[This PR](https://github.com/chanzuckerberg/cryoet-data-portal/pull/647) will add a question to the portal site FAQ and duplicate all of the FAQ content to a new FAQ page on the docsite. Either location can be linked to.
